pyproject.toml

```diff
@@ -96,6 +96,7 @@ dev-dependencies = [
     "requests",
     "sqlalchemy",
     "pandas",
+    "numpy<2.0.0",
```

Comment:
This is temporary, right?

Reply:
Yeah, I just removed this change on my last commit 👍